### PR TITLE
Changes "analyse" to "analyze"

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Just a small consistency fix - in some places, analyze was written as analyse. This fixes that.

## Why It's Good For The Game

Consistency.

## Changelog
:cl:NewSta
spellcheck: changed uses of "analyse" to "analyze"
/:cl:

### CODE REVIEW

**Comment by Dorsisdwarf** [comment]
>:(

**Comment by Dawson1917** [comment]
americans forcefully butcher the english language yet again

**Comment by ATH1909** [comment]
The clear compromise here is to merge this on all of our servers except for Terry.

**Comment by PKPenguin321** [comment]
I made this mistake once too: this is apparently a regional spelling, not a spelling error, and changing it is not allowed under the contributing guidelines

**Comment by ATH1909** [comment]
> I made this mistake once too: this is apparently a regional spelling, not a spelling error, and changing it is not allowed under the contributing guidelines

Is it spelled inconsistently throughout the game, though?

**Comment by PKPenguin321** [comment]
🤷‍♂ heres the thing where its listed if youre curious https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#enforced-not-enforced

File: html/changelogs/.all_changelog.yml [hunk 1]
```diff
@@ -15850,7 +15850,7 @@ DO NOT EDIT THIS FILE BY HAND!  AUTOMATICALLY GENERATED BY ss13_genchangelog.py.
   - tweak: The Scan with Debugger/Device button now reads Copy Ref and no longer sends
       you to the circuit's page when clicked
   - tweak: The assembly's menu is now slightly wider
-  - tweak: The advanced in "integrated advanced medical analyser" is now abbreviated
+  - tweak: The advanced in "integrated advanced medical analyzer" is now abbreviated
```
Comment (ExcessiveUseOfCobblestone) [inline html/changelogs/.all_changelog.yml]: rly

Comment (Tlaltecuhtli) [reply html/changelogs/.all_changelog.yml]: is timetravelling illegal?

**Comment by optimumtact** [comment]
americans can get the goddamn hell out of my codebase

**Comment by optimumtact** [comment]
specifically see https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#enforced-not-enforced

**Pull request closed without merging**